2022/01/20 15:51:57 [emerg] 12316#3372: "upstream" directive is not allowed here in C:\Users\Manuel\Desktop\nginx-1.21.5/conf/nginx.conf:39
2022/01/20 15:52:48 [emerg] 13764#14264: "upstream" directive is not allowed here in C:\Users\Manuel\Desktop\nginx-1.21.5/conf/nginx.conf:39
2022/01/20 15:53:50 [emerg] 804#13704: "upstream" directive is not allowed here in C:\Users\Manuel\Desktop\nginx-1.21.5/conf/nginx.conf:39
2022/01/20 15:55:41 [emerg] 3684#11484: "server" directive is not allowed here in C:\Users\Manuel\Desktop\nginx-1.21.5/conf/nginx.conf:44
2022/01/20 15:56:34 [error] 4300#12816: *1 CreateFile() "C:\Users\Manuel\Desktop\nginx-1.21.5/html/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2022/01/20 16:24:26 [emerg] 12752#436: "upstream" directive is not allowed here in D:\Backend CoderHouse\Desafios Entregables\Desafio entregable 13 Nginx Forker y Cluster\nginx-1.21.5/conf/nginx.conf:26
2022/01/20 16:49:26 [emerg] 5160#13708: invalid number of arguments in "root" directive in D:\Backend CoderHouse\Desafios Entregables\Desafio entregable 13 Nginx Forker y Cluster\nginx-1.21.5/conf/nginx.conf:29
2022/01/20 16:55:25 [emerg] 12348#10216: unexpected "}" in D:\Backend CoderHouse\Desafios Entregables\Desafio 13\nginx-1.21.5/conf/nginx.conf:33
2022/01/20 16:57:12 [emerg] 11344#6308: unexpected "}" in D:\Backend CoderHouse\Desafios Entregables\Desafio 13\nginx-1.21.5/conf/nginx.conf:33
2022/01/20 16:57:40 [emerg] 2012#14248: unexpected "}" in D:\Backend CoderHouse\Desafios Entregables\Desafio 13\nginx-1.21.5/conf/nginx.conf:33
2022/01/20 16:58:43 [emerg] 7548#10932: unexpected "}" in D:\Backend CoderHouse\Desafios Entregables\Desafio 13\nginx-1.21.5/conf/nginx.conf:73
2022/01/20 17:30:10 [error] 4108#14296: *1 directory index of "D:\Backend CoderHouse\Desafios Entregables\Desafio 13\nginx-1.21.5/../node/views/layouts/" is forbidden, client: 127.0.0.1, server: ngnix_node, request: "GET / HTTP/1.1", host: "localhost"
2022/01/20 17:31:59 [error] 4108#14296: *4 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /datos/ HTTP/1.1", upstream: "http://127.0.0.1:8082/datos/", host: "localhost"
2022/01/20 17:33:10 [error] 4108#14296: *16 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /datos/ HTTP/1.1", upstream: "http://127.0.0.1:8082/datos/", host: "localhost"
2022/01/20 17:35:25 [error] 4108#14296: *43 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /datos/ HTTP/1.1", upstream: "http://127.0.0.1:8082/datos/", host: "localhost"
2022/01/20 17:38:44 [error] 4108#14296: *53 directory index of "D:\Backend CoderHouse\Desafios Entregables\Desafio 13\nginx-1.21.5/../node/views/layouts/" is forbidden, client: 127.0.0.1, server: ngnix_node, request: "GET / HTTP/1.1", host: "localhost"
2022/01/20 17:40:02 [error] 4108#14296: *56 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /datos/ HTTP/1.1", upstream: "http://127.0.0.1:8082/datos/", host: "localhost"
2022/01/20 17:48:16 [error] 4108#14296: *63 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /datos/ HTTP/1.1", upstream: "http://127.0.0.1:8082/datos/", host: "localhost"
2022/01/20 17:48:23 [error] 4108#14296: *63 directory index of "D:\Backend CoderHouse\Desafios Entregables\Desafio 13\nginx-1.21.5/../node/views/layouts/" is forbidden, client: 127.0.0.1, server: ngnix_node, request: "GET / HTTP/1.1", host: "localhost"
2022/01/20 17:50:24 [error] 4108#14296: *69 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /datos/ HTTP/1.1", upstream: "http://127.0.0.1:8082/datos/", host: "localhost"
2022/01/20 17:51:00 [error] 4108#14296: *71 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /datos/ HTTP/1.1", upstream: "http://127.0.0.1:8082/datos/", host: "localhost"
2022/01/20 17:56:35 [error] 11612#13564: *19 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /datos/ HTTP/1.1", upstream: "http://127.0.0.1:8081/datos/", host: "localhost"
2022/01/20 17:56:37 [error] 11612#13564: *19 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /datos/ HTTP/1.1", upstream: "http://127.0.0.1:8080/datos/", host: "localhost"
2022/01/20 18:16:17 [error] 1076#2288: *14 directory index of "D:\Backend CoderHouse\Desafios Entregables\Desafio 13\nginx-1.21.5/../node/views/layouts/" is forbidden, client: 127.0.0.1, server: ngnix_node, request: "GET / HTTP/1.1", host: "localhost"
2022/01/21 15:18:28 [emerg] 16160#16164: unknown directive "CONFIGURACION" in D:\Backend CoderHouse\Desafios Entregables\Desafio 13\nginx-1.21.5/conf/nginx.conf:40
2022/01/21 15:18:28 [alert] 14016#2060: worker process 16160 exited with code 1
2022/01/21 15:18:28 [alert] 14016#2060: could not respawn worker
2022/01/21 15:18:36 [emerg] 14396#2428: unknown directive "CONFIGURACION" in D:\Backend CoderHouse\Desafios Entregables\Desafio 13\nginx-1.21.5/conf/nginx.conf:40
2022/01/21 15:19:57 [error] 9340#1336: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /api/random HTTP/1.1", upstream: "http://127.0.0.1:8082/api/random", host: "localhost"
2022/01/21 15:19:59 [error] 9340#1336: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /api/random HTTP/1.1", upstream: "http://127.0.0.1:8083/api/random", host: "localhost"
2022/01/21 15:20:01 [error] 9340#1336: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /api/random HTTP/1.1", upstream: "http://127.0.0.1:8084/api/random", host: "localhost"
2022/01/21 15:20:03 [error] 9340#1336: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /api/random HTTP/1.1", upstream: "http://127.0.0.1:8085/api/random", host: "localhost"
2022/01/21 15:20:16 [error] 9340#1336: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /api/random?cant=10 HTTP/1.1", upstream: "http://127.0.0.1:8085/api/random?cant=10", host: "localhost"
2022/01/21 15:20:18 [error] 9340#1336: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /api/random?cant=10 HTTP/1.1", upstream: "http://127.0.0.1:8084/api/random?cant=10", host: "localhost"
2022/01/21 15:20:20 [error] 9340#1336: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /api/random?cant=10 HTTP/1.1", upstream: "http://127.0.0.1:8083/api/random?cant=10", host: "localhost"
2022/01/21 15:20:22 [error] 9340#1336: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: ngnix_node, request: "GET /api/random?cant=10 HTTP/1.1", upstream: "http://127.0.0.1:8082/api/random?cant=10", host: "localhost"
